README.md

```diff
@@ -30,3 +30,25 @@ that belong in the OEM partition. Any RDEPENDS from these packages should
 be copied to the RDEPENDS in `board-packages` to ensure they are built.
 
 `coreos-base/coreos-oem-*` are metapackages for OEM specific ACIs. 
+
+# Updating
+
+To update follow the following steps:
+
+* Remove or rename the whole folder of the package to prepare the import from
+  upstream Gentoo, not only resetting the ebuild file but also any additional
+  files like patches or downstream additions under `files`.
+* Run `~/trunk/src/scripts/update_ebuilds --portage_stable . CATEGORY/PACKAGE`
+  in the `coreos-overlay` folder to import a new version from upstream Gentoo.
```

Comment:
I admit I never used that script. What I have is I have my own clone for gentoo repo and at this point I do `cp -a "${GENTOO_REPO}/{CATEGORY}/${PACKAGE}" "${CATEGORY}/${PACKAGE}"`, but I guess the script is going to do more or less the same.

But what I do next is that I drop all the ebuilds I don't need, so I only have one ebuild. And I drop all the patches and stuff from the `files` directory that are unused. Having multiple ebuilds is just confusing, especially that we often end up patching only one of them.

Reply:
Yes, will mention that unused ebuild files should be dropped.